### ISSUE TITLE
Support Content-ID and Message-ID URLs

### ISSUE THREAD

**Issue by jhillyerd** [jhillyerd/inbucket#71]
To render images embedded in HTML messages, Inbucket needs to support Content-ID and Message-ID URLs.

https://tools.ietf.org/html/rfc2392

**Comment by jhillyerd** [comment]
On hold until the outcome of https://github.com/microcosm-cc/bluemonday/issues/58 is known.

**Comment by jhillyerd** [comment]
Possibly unblocked by https://github.com/microcosm-cc/bluemonday/pull/179

**Comment by nasyarobby** [comment]
Hi, is there any updates on this? Recently we switched to inbucket and just realized that inlined images dont show up.

**Comment by jhillyerd** [comment]
Unfortunately it's a significant amount of work to implement, since it requires parsing HTML to rewrite URL attributes, which may be a rabbit hole in itself. I'm not currently working on it.
